src/cmd/cmd_conc.ml

```diff
@@ -490,14 +490,22 @@ let cmd ~profiling ~debug ~unsafe ~rac ~srac ~optimize ~workers:_
     else Ok ()
   in
   if print_paths then Fmt.pr "Completed paths: %d@." !count_path;
+  let to_file path assignements =
+    match Bos.OS.File.write path (to_string assignements) with
+    | Ok _ -> ()
+    | Error (`Msg msg) ->
+      Fmt.epr "Failed to write model to the given path: %s@." msg
```

Comment:
I think it is better to let owi fail when it can not write to the file. (meaning, this function should return a Result)